app/move/controllers/create/assessment.js

```diff
@@ -12,18 +12,26 @@ class AssessmentController extends CreateBaseController {
       const questions = await referenceDataService.getAssessmentQuestions(
         assessmentCategory
       )
+      const implicitQuestions = questions.filter(
+        fieldHelpers.extractItemsForImplicitFields.bind(null, fields)
+      )
 
-      const implicitField = createFields.assessmentCategory(assessmentCategory)
-      implicitField.items = questions
-        .filter(fieldHelpers.extractItemsForImplicitFields.bind(null, fields))
-        .map(fieldHelpers.mapAssessmentQuestionToConditionalField)
-        .map(fieldHelpers.mapAssessmentQuestionToTranslation)
-        .map(fieldHelpers.mapReferenceDataToOption)
+      if (implicitQuestions.length) {
+        const implicitField = createFields.assessmentCategory(
+          assessmentCategory
+        )
 
-      req.form.options.fields[assessmentCategory] = implicitField
+        req.form.options.fields[assessmentCategory] = {
+          ...implicitField,
+          items: implicitQuestions
+            .map(fieldHelpers.mapAssessmentQuestionToConditionalField)
+            .map(fieldHelpers.mapAssessmentQuestionToTranslation)
+            .map(fieldHelpers.mapReferenceDataToOption),
+        }
+      }
```

Comment:
Maybe this block of code that populates `assessmentCategory` should go into its own method -- or possibly just the block that builds `items`.

Reply:
I was kind of thinking of refactoring but didn't want to go down a rabbit hole just yet after we spent a fair bit of time on it last time. 

The main aim was to ensure it didn't create the implicit field if there were no implicit questions for that step.

Reply:
I did start a refactor to try and remove the need for the `__yesno` suffix being required between a few files so I think we can revisit that later.

Reply:
👍